ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.hbs

```diff
@@ -12,23 +12,34 @@
  **/
  --}}
 <div class="metacard-interaction interaction-copy-coordinates" data-help="Copies the coordinates to your clipboard."
-    data-clipboard-text="{{this.clickLat}} {{this.clickLon}}">
+     data-clipboard-text="{{this.clickLat}} {{this.clickLon}}">
     <div class="interaction-icon fa fa-clipboard">
     </div>
     <div class="interaction-text">
-        Copy Coordinates to Clipboard
+        Copy Coordinates in Decimal Degrees (DD)
         <div>
             <span>{{this.clickLat}}</span>
             <span>{{this.clickLon}}</span>
         </div>
     </div>
 </div>
+<div class="metacard-interaction interaction-copy-dms" data-help="Copies the DMS coordinates to your clipboard."
+     data-clipboard-text="{{this.clickDms}}">
+    <div class="interaction-icon fa fa-clipboard">
+    </div>
+    <div class="interaction-text">
+        Copy Coordinates in Degrees Minutes Seconds (DMS)
+        <div>
+            <span>{{this.clickDms}}</span>
+        </div>
+    </div>
+</div>
 <div class="metacard-interaction interaction-copy-wkt" data-help="Copies the WKT of the coordinates to your clipboard."
-    data-clipboard-text="POINT ({{this.clickLon}} {{this.clickLat}})">
+     data-clipboard-text="POINT ({{this.clickLon}} {{this.clickLat}})">
     <div class="interaction-icon fa fa-clipboard">
     </div>
     <div class="interaction-text">
-        Copy WKT Coordinates to Clipboard
+        Copy Coordinates in Well Known Text (WKT)
```

Comment:
`Copy Coordinates as Well Known Text (WKT)`??

Reply:
@stustison I used values Leah gave me. Are you saying it should be `as` instead of `in`?

Reply:
`as` sounds more correct to me for both of these (decimal and wkt)

Reply:
@lcrosenbu Would you like me to change it to `as` for all of them?

Reply:
`as` is great.